sample-code/src/langchain-orchestration.ts

```diff
@@ -110,3 +111,73 @@ export async function invokeChainWithOutputFilter(): Promise<string> {
       }
     });
 }
+
+/**
+ * Trigger masking the input provided to the large language model.
+ * @returns The answer from ChatGPT.
+ */
+export async function invokeChainWithMasking(): Promise<string> {
+  const orchestrationConfig: OrchestrationModuleConfig = {
+    // define the language model to be used
+    llm: {
+      model_name: 'gpt-4o'
+    },
+    // define the template
+    templating: {
+      template: [
+        {
+          role: 'user',
+          content: 'Summarize the following CV in 10 sentences: {{?orgCV}}'
+        }
+      ]
+    },
+    masking: {
+      masking_providers: [
+        buildDpiMaskingProvider({
+          method: 'pseudonymization',
```

Comment:
[pp] Would it be better to use anonymization since user will be able to see the result in the browser?

```suggestion
          method: 'anonymization',
```